spec/send_sms_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'send_sms'
+require 'order'
+
+describe SendSMS do
+  let(:confirmation) { Order.new(menu.new) { include SendSMS } }
```

Comment:
There is a hard dependency on Order here, but menu has been doubled. Recommend trying to use doubles for all dependencies.